backend/frontend/templates/frontend/auth/page_login.html

```diff
@@ -0,0 +1,30 @@
+{% extends "frontend/auth/base.html" %} {% block content %}
+<div class="mx-auto w-1/3 p-6 bg-white rounded">
+  <h1 class="{{ styles.heading1 }}">Login</h1>
+  <hr class="my-2" />
+  {% if messages %}
```

Comment:
Hmm, would this not be better in a more foundational base template? Then all pages will have message functionality.  Eg I'm sure it would make sense to have messages when a user adds a review to a card, updates their user profile or any number of other things. Or do you think that kind of thing will be displayed completely differently? 

I'm not actually sure at this point.